### DISCUSSION TITLE
Badges in credly

### DISCUSSION THREAD

**Discussion by YuriyMorozyuk95** [microsoft/hack-together-teams#148] [Q&A]
Hi, I have a quick question. Do the winner of hackathon receive different badges in credly?

**Comment by AnthonyByansi** [comment]
It is the same digital Credly badge, with prizes attached to that!

**Comment by waldekmastykarz** [comment]
Like @AnthonyByansi mentioned, everyone who submitted a valid hack, will get the same badge.
